_DENA/scripts/run_ground_truthing.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
Could we put import os below import glob? Trying to stick with pep8 alphabetical import standards